src/manifest.json

```diff
@@ -1,22 +1,16 @@
 {
-	"name": "500pxOnNewTab",
+	"name": "__MSG_extensionName__",
 	"version": "1.0",
 	"description": "Apply an new amazing wallpaper from the 500px website when you want.",
-	"author": {
-    "name": "Rom",
-    "email": "skav@gmx.fr",
-    "url": "https://github.com/xerta555/"
-	},
+    "author": "Rom",
+    "homepage_url": "https://github.com/xerta555/",
 	"default_locale": "en",
-	"bugs": {
-    "email": "skav@gmx.fr"
-	},
 	"applications": {
 		"gecko": {
-			"id": "https://github.com/xerta555/",
+			"id": "500pxOnNewTab@xerta555.github.io",
```

Comment:
The ID must be a GUID or an email-like address. It does not need to be a real email (to prevent spam).

Reply:
Noted.